core/math/vector3.h

```diff
@@ -84,6 +84,29 @@ struct _NO_DISCARD_ Vector3 {
 		return Vector3(MAX(x, p_vector3.x), MAX(y, p_vector3.y), MAX(z, p_vector3.z));
 	}
 
+	_FORCE_INLINE_ Vector3 rotate_toward(Vector3 p_to, real_t p_delta) const {
+		if (Math::is_zero_approx(p_delta)) {
+			return *this;
```

Comment:
Is this check necessary because of some singularity I'm not seeing? A delta of zero would be an uncommon value, so I would prefer to not make the happy code path slower in order to handle this case.

Reply:
The function was written with some help on the contributor chat and I think this was supposed to protect against dividing by zero before angle < p_delta was thought of, I don't think it's needed so I'll take it out.